de/manual/ch_Tools_Assistants.xml

```diff
@@ -70,43 +70,39 @@
       <para>Die Funktion <guilabel>Buchungen suchen</guilabel> wird verwendet, um nach Buchungen in Ihrem
         aktuellen &app;-Buch zu suchen und die Ergebnisse in einer Registerkarte vom Typ
```

Comment:
`in einem meist neuen Reiter vom Typ` wäre mein Vorschlag für die Vereinheitlichung. Kannst du das  `meist` vielleicht noch präzisieren?